yt_dlp/extractor/niconico.py

```diff
@@ -192,20 +190,38 @@ def _perform_login(self, username, password):
         self._request_webpage(
             'https://account.nicovideo.jp/login', None,
             note='Acquiring Login session')
-        urlh = self._request_webpage(
+        resp_page = self._request_webpage(
             'https://account.nicovideo.jp/login/redirector?show_button_twitter=1&site=niconico&show_button_facebook=1', None,
             note='Logging in', errnote='Unable to log in',
             data=urlencode_postdata(login_form_strs),
             headers={
                 'Referer': 'https://account.nicovideo.jp/login',
                 'Content-Type': 'application/x-www-form-urlencoded',
-            })
-        if urlh is False:
-            login_ok = False
-        else:
-            parts = compat_urllib_parse_urlparse(urlh.geturl())
-            if compat_parse_qs(parts.query).get('message', [None])[0] == 'cant_login':
-                login_ok = False
+            }).read().decode('utf-8')
+        need_mfa = 'oneTimePw' in resp_page
+        if need_mfa:
+            mfa_submit = self._search_regex(
+                r'(?<=<form action=["\'])(.*?)(?=["\'])', resp_page, 'mfa_submit',
```

Comment:
```suggestion
                r'<form action=["\']([^"\'])["\']', resp_page, 'mfa_submit',
```
why use lookahead/behind?